libarchive/ffi.py

```diff
@@ -125,7 +125,7 @@ def ffi(name, argtypes, restype, errcheck=None):
 ffi('entry_rdevmajor', [c_archive_entry_p], c_uint)
 ffi('entry_rdevminor', [c_archive_entry_p], c_uint)
 
-ffi('entry_set_size', [c_archive_entry_p, c_int], c_int)
+ffi('entry_set_size', [c_archive_entry_p], c_longlong)
```

Comment:
Ok, thank you- it is my first time I use ctypes module. What tests do you think will help to avoid such mistakes?